src/views/activityBar/changes/changeTreeView/fileTreeView.ts

```diff
@@ -123,12 +123,17 @@ export class FileTreeView implements TreeItemWithoutChildren {
 			newContent,
 			oldContent,
 		});
+
+		const tabTitle = `${file.filePath} ${
+			patchsetBase?.number ?? 'Base'
+		} -> ${file.currentRevision.number}`;
```

Comment:
Can we maybe put in something like "latest" or current if it's the latest patchset? You can find out the latest patchset by using `file.currentRevision.isCurrentRevision`.